src/gigs-board/components/layout/Controls.jsx

```diff
@@ -45,141 +45,17 @@ function href(widgetName, linkProps) {
-                aria-controls={`collapseSubmissionEditorNavbar`}
-              >
-                <i class="bi-rocket-fill"> </i>
-                Solution
-              </a>
-            </li>
-            <li class="nav-item">
-              <a
-                class="nav-link active"
-                data-bs-toggle="collapse"
-                href={`#collapseAttestationEditorNavbar`}
-                role="button"
-                aria-expanded="false"
-                aria-controls={`collapseAttestationEditorNavbar`}
-              >
-                <i class="bi-check-circle-fill"> </i>
-                Attestation
-              </a>
-            </li>
-            <li class="nav-item">
-              <a
-                class="nav-link active"
-                data-bs-toggle="collapse"
-                href={`#collapseSponsorshipEditorNavbar`}
-                role="button"
-                aria-expanded="false"
-                aria-controls={`collapseSponsorshipEditorNavbar`}
-              >
-                <i class="bi-cash-coin"> </i>
-                Sponsorship
-              </a>
-            </li>
-          </ul>
-        </div>
-      </div>
-    </div>
-    <div class="row" id={`accordionNavbar`}>
-      <div
-        class="collapse"
-        id={`collapseCommentEditorNavbar`}
-        data-bs-parent={`#accordionNavbar`}
-      >
-        {widget("components.posts.PostEditor", {
-          postType: "Comment",
-          parentId: null,
-          labels: props.labels,
-        })}
-      </div>
-      <div
-        class="collapse"
-        id={`collapseIdeaEditorNavbar`}
-        data-bs-parent={`#accordionNavbar`}
-      >
-        {widget("components.posts.PostEditor", {
-          postType: "Idea",
-          parentId: null,
-          labels: props.labels,
-        })}
-      </div>
-      <div
-        class="collapse"
-        id={`collapseSubmissionEditorNavbar`}
-        data-bs-parent={`#accordionNavbar`}
-      >
-        {widget("components.posts.PostEditor", {
-          postType: "Submission",
-          parentId: null,
-          labels: props.labels,
-        })}
-      </div>
-      <div
-        class="collapse"
-        id={`collapseAttestationEditorNavbar`}
-        data-bs-parent={`#accordionNavbar`}
-      >
-        {widget("components.posts.PostEditor", {
-          postType: "Attestation",
-          parentId: null,
-          labels: props.labels,
-        })}
-      </div>
-      <div
-        class="collapse"
-        id={`collapseSponsorshipEditorNavbar`}
-        data-bs-parent={`#accordionNavbar`}
-      >
-        {widget("components.posts.PostEditor", {
-          postType: "Sponsorship",
-          parentId: null,
-          labels: props.labels,
-        })}
-      </div>
-    </div>
+  <div class="d-flex flex-row-reverse">
+    <a
+      class="btn btn-light"
+      style={{
+        backgroundColor: "#0C7283",
+        color: "#f3f3f3",
+      }}
+      href="gigs-board.pages.Create"
```

Comment:
Use `href` helper (see https://github.com/near/neardevhub-widgets/pull/122#discussion_r1198920006 for more details)